### ISSUE TITLE
ps-key-listener, ps-click-outside re-open issue

### ISSUE THREAD

**Issue by resistancecanyon** [dpiccone/ng-pageslide#133]
If we set either ps-keylistener or ps-click-outside to true, the slider is working properly & closes the first time.
The second time we try to open the slider by setting 
`ps-open="scopeVariable"` & setting `$scope.scopeVariable=true;`
The slider won't open again.
The scope variable is not affected anymore.

I had to made it work by using manual ng-click on a button to explicitly set the `$scope.scopeVariable` to `true` & `false`

> version : ng-pageslide#^2.0.1

**Comment by songokudbz** [comment]
I can't make it work! How did you fixed this?

**Comment by songokudbz** [comment]
I've fixed this by adding $parent: <div pageslide ps-open="$parent.isMenuOpened"... . In my case i was including the template with ng-include, so I needed to add $parent. Hope it helps.

**Comment by dpiccone** [comment]
I am trying to replicate it, but I am not able to.
There's an example with the key listener, and it keeps working when you use the key listener to close and the button (which flips `$scope.open` bound to `ps-open`) to true.
Could you put together a plunker for this?

**Comment by songokudbz** [comment]
Yes I will try to put one together when I have a little time. Seasons greetings until then.

**Comment by dpiccone** [comment]
Likewise :christmas_tree: :christmas_tree: :christmas_tree: :)
